### PR TITLE
Add argparse to pass in a juju controller:model

### PR DESCRIPTION
**Pull request by battlemidget**
Needed the ability to pass in juju controller:model context rather than relying on juju switch or the current environment in CI tests.

Also did some pep8 formatting

Signed-off-by: Adam Stokes <battlemidget@users.noreply.github.com>

### CODE REVIEW

File: collect.py [hunk 1]
```diff
@@ -71,43 +75,66 @@ def store_results(temppath):
...
-    command(temppath, 'kubernetes-worker-config', 'juju config kubernetes-worker --format yaml')
-    command(temppath, 'kubeapi-load-balancer-config', 'juju config kubeapi-load-balancer --format yaml')
-    command(temppath, 'etcd-config', 'juju config etcd --format yaml')
-    command(temppath, 'easyrsa-config', 'juju config easyrsa --format yaml')
-    command(temppath, 'flannel-config', 'juju config flannel --format yaml')
+    command(temppath, 'status', 'juju status {} --format yaml'.format(model))
+    command(temppath, 'debug-log', 'juju debug-log {} --replay'.format(model))
+    command(temppath, 'model-config', 'juju model-config {}'.format(model))
+    command(temppath, 'controller-debug-log',
+            'juju debug-log {} --replay'.format(model))
```
Comment (kwmonroe) [inline collect.py]: I don't think you want the user's model arg here.  The controller debug-log needs to be `-m controller`.

File: collect.py [hunk 2]
```diff
@@ -71,43 +75,69 @@ def store_results(temppath):
...
-    command(temppath, 'kubernetes-worker-config', 'juju config kubernetes-worker --format yaml')
-    command(temppath, 'kubeapi-load-balancer-config', 'juju config kubeapi-load-balancer --format yaml')
-    command(temppath, 'etcd-config', 'juju config etcd --format yaml')
-    command(temppath, 'easyrsa-config', 'juju config easyrsa --format yaml')
-    command(temppath, 'flannel-config', 'juju config flannel --format yaml')
+    command(temppath, 'status', 'juju status {} --format yaml'.format(model))
+    command(temppath, 'debug-log', 'juju debug-log {} --replay'.format(model))
+    command(temppath, 'model-config', 'juju model-config {}'.format(model))
+    command(temppath, 'controller-debug-log',
+            'juju debug-log {} --replay'.format(model.split(':')[0]))
```
Comment (kwmonroe) [inline collect.py]: This isn't quite right... The model name for the controller is literally `controller`.  So we don't want `model[0]`, we want the string `controller`, like:
```
command(temppath, 'controller-debug-log', 'juju debug-log -m controller --replay')
```

**Comment by battlemidget** [comment]
Now @kwmonroe has just made stokachu :-1: ill fix this tomorrow

**Comment by kwmonroe** [comment]
I thought we were bonding.  Can't wait to see what you come up with manana!

**Pull request closed without merging**